### PR TITLE
Layout change to conform to Arduino IDE requirements

### PR DESCRIPTION
**Pull request by fiveangle**
-Renamed "src" dir to "openLRSng" and associated path references
-Minor fix for leonardo description in PIO config
-Key source directory off of platformio.ini config file for build scripts
-Readme to root directory

### CODE REVIEW

File: buildroot/bin/backup_configs [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-cp src/openLRSng.ino     src/openLRSng.default
+cp openLRSng/openLRSng.ino     openLRSng/openLRSng.default
```
Comment (mikeller) [inline buildroot/bin/backup_configs]: Looking at this, would it make sense to create a config file for pathing and stuff, and include it at the top of all of the scriptlets?

Comment (fiveangle) [reply buildroot/bin/backup_configs]: We could but it's not really to the tipping point where it saves much effort.  How about maybe tackle that on the auto-deploy Travis revamp ? :)

Comment (mikeller) [reply buildroot/bin/backup_configs]: Fair enough.

**Comment by csurf** [comment]
@fiveangle 
side-note, the README is still in the src directory. It should go in the repo root...

**Comment by csurf** [comment]
@fiveangle cool... don't forget to squash the PR commits into a single commit to keep the tree clean ;-) 👍

**Comment by fiveangle** [comment]
yeah, if you see multiple commits, take that as a sign that i'm still tinkering ;)

this one should be good to go

File: buildroot/bin/backup_configs (hunk 1 above)
Comment (fiveangle) [reply buildroot/bin/backup_configs]: why not 🤷‍♂️

**Pull request merged**